gatsby-site/src/components/Hero.js

```diff
@@ -0,0 +1,108 @@
+import React from "react"
+import PropTypes from "prop-types"
+import { css } from "styled-components"
+
+const Hero = ({ dense, background, hubLogo, hubName, hubWebsite }) => {
+  return (
+    <section
+      css={css`
```

Comment:
I am not familiar with this type of pattern so I could be missing information here.

But is there a way we could take the css out of this return section?
I personally feel like it makes it really hard to tell what is really making up this Hero Component.
Just a suggestion!

Reply:
Done! I just did the style I'm used to. I have no preference for which one we use so happy to switch to the one that works best for you.